ldap/servers/slapd/back-ldbm/ldbm_entryrdn.c

```diff
@@ -1570,12 +1576,15 @@ _entryrdn_get_elem(entryrdn_db_ctx_t *ctx,
                                          key->data, data->size, data->ulen, rc);
         }
         _ENTRYRDN_DEBUG_GOTO_BAIL();
+        slapi_ch_free((void**)elem);
```

Comment:
This will prevent the log of the rdn with the error 
   slapi_log_err(SLAPI_LOG_TRACE, "_entryrdn_get_elem", "<-- _entryrdn_get_elem (*elem rdn=%s) rc=%d\n",
                      RDN_ADDR(*elem), rc); 
 IMHO we should rather call if (rc) { slapi_ch_free((void**)elem); } after login the error in the bail section.